.github/workflows/publish-docker-images.yaml

```diff
@@ -66,8 +66,18 @@ jobs:
         uses: docker/build-push-action@v5
         with:
           context: .
-          platforms: linux/amd64,linux/arm64
+          platforms: ${{ github.event_name != 'pull_request' && 'linux/amd64,linux/arm64' || 'linux/amd64' }}
```

Comment:
Since we don't cross-build on PRs, does that mean we can also skip qemu and build on PRs?